controllers/referee/referee.py

```diff
@@ -65,63 +65,63 @@ def run(self, CI):
         self.setLabel(1, '█' * 100, 0, 0.048, 0.1, 0xffffff, 0.3, 'Lucida Console')
         self.setLabel(2, participant, 0.01, 0.003, 0.08, 0xff0000, 0, 'Arial')
         self.setLabel(3, opponent, 0.01, 0.051, 0.08, 0x0000ff, 0, 'Arial')
-        ko_labels = ['', '']
-        coverage_labels = ['', '']
+        ko_labels = [''] * 2
+        coverage_labels = [''] * 2
         while True:
-            if time % 200 == 0:
+            if time % (10 * time_step) == 0:
```

Comment:
Here we should update the time on the ring every second instead of every 200 ms
```suggestion
            if time % (1000) == 0:
```